pkg/modfilter/modfilter.go

```diff
@@ -0,0 +1,192 @@
+package modfilter
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/gomods/athens/pkg/config/env"
+)
+
+var (
+	pathSeparator = "/"
+)
+
+// ModFilter is a filter of modules
+type ModFilter struct {
```

Comment:
can you name this `Filter`? that way, the type will read `mod.Filter` when it's used - instead of the redundant `mod.ModFilter`